config/kibana.yml

```diff
@@ -1,3 +1,4 @@
+csp.strict: false
```

Comment:
Is this necessary for IE11? If so we shouldn't change the kibana.yml config file. I think we might be able to specify it as a command line flag in the `config.ie.js` functional test runner config file.

Reply:
This was a necessary decision to prevent legacy browsers by default and we should not be circumventing this. https://github.com/elastic/kibana/pull/41700

If this is necessary for tests, we could be setting it through the functional test runner.

Reply:
👍 similar to firefox, https://github.com/elastic/kibana/blob/6b3430ef9cbf305e2b022bdd4329258ee3118ee9/test/functional/config.firefox.js#L24.  we can add `serverArgs:.

Reply:
I addressed this, but have some questions if the kibana server args should be moved from the new `config.ie.js` to the base `config.js`.  Where it is now in `config.ie.js` means you have to use the --config option and specify that config when starting functional_tests_server and when running functional_test_runner.